UKHomeOffice/cerberus-service
# COP-7227: Create Cron Job & schedule automation tests to run against DEV environment

## Description
This PR has changes to Create Cron Job & schedule automation tests to run against DEV environment.
Tests schedule to run every 1 hour between 8am to 5 pm
Report & test status slacked to #cop-test-report channel

## To Test
execute the following command 
kubectl --context=acp-notprod_COP --namespace=cop-cerberus-dev  get cronjobs
you should see the created cronjob
`cerberus-functional-tests`
check #cop-test-report for the report and test status
## Developer Checklist

\* Required
